projects/common/lib/components/consent-modal/consent-modal.component.ts

```diff
@@ -0,0 +1,177 @@
+import { forwardRef, Component, EventEmitter, Input, Output, ViewChild, Inject} from '@angular/core';
+import * as moment from 'moment';
+import {ModalDirective} from 'ngx-bootstrap';
+import {ApplicationBase} from '../../../models/src/application-base.model';
+import { Observable, of } from 'rxjs';
+import { HttpClient, HttpHeaders,HttpErrorResponse } from '@angular/common/http';
+import { Response } from '@angular/http';
+import { CommonLogger, CommonLogMessage } from '../../../services/src/logger.service';
+import {AbstractHttpService} from "../../../services/src/abstract-api-service"
+import { ControlContainer, ControlValueAccessor, NgForm, NG_VALUE_ACCESSOR } from '@angular/forms';
+
+
+
+/**
+ * Consent Modal is a Modal with the Information or Notice. It can be used to get the User's consent an 
+ * then proceed with the application. It also makes an API call to the SPA-ENV server to see if the app is under 
+ * maintenance.
+ * 
+ * 
+ * @example
+*       	<common-consent-modal #mspConsentModal [body]="'Body Of Consent'" 
+*               [title]="'Notice'" [application]="mspAccountApp" 
+*               [processName]="'MSP'"
+*               [agreeLabel]="'I have read and understand this info'" 
```

Comment:
Same point about `label=` vs `[label]=`

Reply:
Also applies to [processName] and [title], can be without [] and extra "s

Reply:
In general just keep an eye out for removing any data binding for string literals.